src/test/java/seedu/linkedout/logic/parser/ViewCommandParserTest.java

```diff
@@ -24,11 +22,11 @@ public void parse_emptyArg_throwsParseException() {
     public void parse_validArgs_returnsViewCommand() {
         // no leading and trailing whitespaces
         ViewCommand expectedViewCommand =
-                new ViewCommand(new NameContainsKeywordsPredicate(Arrays.asList("Alice", "Bob")));
+                new ViewCommand(new NameContainsAllKeywordsPredicate("Alice Bob"));
         assertParseSuccess(parser, "Alice Bob", expectedViewCommand);
 
-        // multiple whitespaces between keywords
-        assertParseSuccess(parser, " \n Alice \n \t Bob  \t", expectedViewCommand);
+        //case insensitive
```

Comment:
add whitespace between case and //, just to standardise 
```suggestion
        // case insensitive
```